be/src/vec/functions/function_jsonb.cpp

```diff
@@ -1097,10 +1097,15 @@ struct JsonbLengthUtil {
                 }
             }
             auto jsonb_value = jsonb_data_column->get_data_at(i);
+            if (UNLIKELY(jsonb_value.size == 0)) {
+                null_map->get_data()[i] = 1;
+                res->insert_data(nullptr, 0);
+                continue;
```

Comment:
please add the test case in the issue to regression test

Reply:
updated. added regression test case.